frameworks/JavaScript/uwebsockets.js/src/utils.js

```diff
@@ -57,3 +59,29 @@ export function escape(text) {
   if (unsafeHTMLMatcher.test(text) === false) return text;
   return text.replace(unsafeHTMLMatcher, function (m) { return escapeHTMLRules[m] || m; });
 }
+
+/**
+ * Using Slow json stringify module to get faster results as explained in just-js blog post
```

Comment:
Could you please remove the phrase "as explained in just-js blog post" and add the link to the blog post in the PR comment?

Reply:
Sure will add it. Meanwhile this is the post i was referring to. 
https://just.billywhizz.io/blog/on-javascript-performance-01/